consoles/VNC.pm

```diff
@@ -159,7 +164,7 @@ sub login {
 }
 
 sub _handshake_protocol_version {
-    my $self = shift;
+    my ($self) = @_;
```

Comment:
aha! so *you* prefer to also replace the single argument assignment shift's with `@_`…